packages/firebase_vertexai/firebase_vertexai/lib/src/vertex_version.dart

```diff
@@ -13,4 +13,4 @@
 // limitations under the License.
 
 /// version number for the package, should be align with pubspec.yaml.
-const packageVersion = '0.1.0';
+const packageVersion = '0.2.0';
```

Comment:
Nothing in this PR looks breaking to me - we could bump to `0.1.1` if we don't have any other breaking changes.

Reply:
I think the 0.2.0 is introduced when the whole flutterfire has breaking change. This file should be updated automatically in the future for release. Currently resolved to use the existing file in the default branch